lib/app/mailer.ex

```diff
@@ -0,0 +1,3 @@
+defmodule App.Mailer do
```

Comment:
given that we aren't _using_ mailer in this app, could we just delete this file? ✂️

Reply:
Do you want me to delete these files? Why did you merge if you gave this feedback?